_episodes/08-crop-raster-data.md

```diff
@@ -413,8 +371,7 @@ cropped_raster.plot.imshow(figsize=(8,8))
 
```

Comment:
For the command above, do we need to add an image for the visualise?

Reply:
Thx for pointing this out! In fact, after reading this section again, I realized that I was doing the previous reprojection in another way, which is causing the two images here in the same CRS instead of different CRS as stated in the text. I updated the intro text and added a new image. Would you like to give it another look?

Reply:
Hey, I tried to take a look at the image but I cannot see the figure. Is the file pushed?

Reply:
You are right, I forgot to push the image. It should be there now.